### PR TITLE
Removing project that contains no tests due to being only an interface.

### PR DESCRIPTION
**Pull request by Chrisboh**
@bartonjs 

Currently we build this DLL and run it but it contains no tests and never will as everything in System.Security.Principal is an interface or enum. #854

### CODE REVIEW

File: src/System.Security.Principal/System.Security.Principal.sln [hunk 1]
```diff
@@ -1,12 +1,10 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22823.1
+VisualStudioVersion = 14.0.24720.0
```
Comment (weshaggard) [inline src/System.Security.Principal/System.Security.Principal.sln]: Don't change the VS version :(

**Comment by stephentoub** [comment]
Test Innerloop Ubuntu Release Build and Test please (Jenkins failure)

**Comment by weshaggard** [comment]
LGTM

File: src/System.Security.Principal/System.Security.Principal.sln (hunk 1 above)
Comment (stephentoub) [inline src/System.Security.Principal/System.Security.Principal.sln]: Does the VS version actually matter for anything impactful?  I keep reverting this change whenever VS does it, but it's so insisting, I wouldn't mind just letting it have its way :)

**Comment by stephentoub** [comment]
LGTM2

File: src/System.Security.Principal/System.Security.Principal.sln (hunk 1 above)
Comment (weshaggard) [inline src/System.Security.Principal/System.Security.Principal.sln]: The version number update is fine. @Chrisboh was in my office when I looked at this and I was just having fun with him.

In general I don't worry about these VS version number updates in the solution files. I don't think we should take PR's to just update those but if someone is updating the solution for another reason then we should just update these versions with them.

Comment (stephentoub) [inline src/System.Security.Principal/System.Security.Principal.sln]: Ok :smile:

**Comment by stephentoub** [comment]
Test Innerloop Ubuntu Release Build and Test please

**Comment by Chrisboh** [comment]
Thanks guys!

**Pull request merged**